schema/release-schema.json

```
@@ -20,7 +20,7 @@
     },
     "date": {
       "title": "Release Date",
-      "description": "The date on which the information contained in the release was first recorded in, or published by, any system. For a compiled release, the maximum `date` among the individual releases used to create the compiled release.",
```

The part about the compiled release is not a full sentence - it's the alternative definition in case of a compiled release. So, I think it has to appear before the full sentence about uniqueness.

```suggestion
      "description": "The date on which the information contained in the release was first recorded in, or published by, any system. For a compiled release, the maximum `date` among the individual releases used to create the compiled release. The release date should be unique within the scope of the contracting process identified by the Open Contracting ID (ocid), for a given version of OCDS.",
```